elasticsearch_dsl/field.py

```diff
@@ -114,17 +118,30 @@ class Object(Field):
     name = 'object'
     _coerce = True
 
-    def __init__(self, doc_class=None, **kwargs):
-        self._doc_class = doc_class
+    def __init__(self, doc_class=None, dynamic=None, properties=None, **kwargs):
+        """
+        :param document.InnerDoc doc_class: base doc class that handles mapping.
```

Comment:
please use `:arg ...` instead of `:param ....` for consistency throughout the library